source/routing/tests/frequency_raptor_test.cpp

```diff
@@ -427,3 +427,217 @@ BOOST_AUTO_TEST_CASE(freq_vj_end_time_is_smaller_than_start_time) {
     BOOST_REQUIRE_EQUAL(res.size(), 0);
 
 }
+
+BOOST_AUTO_TEST_CASE(freq_vj_with_boarding_alighting) {
+    ed::builder b("20170101");
+    b.frequency_vj("A", "8:00"_t, "18:00"_t, "00:30"_t)
+        ("stop1", "8:02"_t, "8:00"_t, std::numeric_limits<uint16_t>::max(), false, true, 0, 300)
+        ("stop2", "8:07"_t, "8:05"_t, std::numeric_limits<uint16_t>::max(), true, true, 900, 900)
+        ("stop3", "8:12"_t, "8:10"_t, std::numeric_limits<uint16_t>::max(), true, false, 300, 0);
+
+    b.data->pt_data->index();
+    b.finish();
+    b.data->build_raptor();
+    b.data->build_uri();
+    RAPTOR raptor(*(b.data));
+
+    auto result = raptor.compute(
+        b.data->pt_data->stop_areas_map["stop1"], b.data->pt_data->stop_areas_map["stop3"],
+        "07:58"_t, 0, DateTimeUtils::inf, type::RTLevel::Base, 2_min
+    );
+
+    BOOST_REQUIRE_EQUAL(result.size(), 1);
+    result.at(0).print();
+    BOOST_REQUIRE_EQUAL(result.at(0).items.size(), 3);
+    BOOST_CHECK_EQUAL(result.at(0).items[0].type, ItemType::boarding);
+    BOOST_CHECK_EQUAL(result.at(0).items[0].departure, boost::posix_time::time_from_string("2017-Jan-01 08:25:00"));
```

Comment:
(i don't know if it's possible, if you haven't tried yet) can you do `"20170101T082500"_dt`?